extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -15,19 +15,24 @@
  */
 
 import {CSS} from '../../../build/amp-subscriptions-0.1.css';
+import {Entitlement} from './entitlement';
 import {EntitlementStore} from './entitlement-store';
-import {Entitlements} from '../../../third_party/subscriptions-project/apis';
 import {LocalSubscriptionPlatform} from './local-subscription-platform';
 import {PageConfig, PageConfigResolver} from '../../../third_party/subscriptions-project/config';
 import {Renderer} from './renderer';
+import {ServiceAdapter} from './service-adapter';
 import {SubscriptionPlatform} from './subscription-platform';
+import {dev, user} from '../../../src/log';
+import {dict} from '../../../src/utils/object';
 import {installStylesForDoc} from '../../../src/style-installer';
 import {tryParseJson} from '../../../src/json';
-import {user} from '../../../src/log';
 
 /** @const */
 const TAG = 'amp-subscriptions';
 
+/** @typedef {{loggedIn: boolean, subscribed: boolean, granted: boolean, entitlement: !Entitlement}} */
+export let RenderState;
```

Comment:
We should instead create a small class with the `json()` method. Btw, for rendering, `entitlement` should be output in json also with `json()`. E.g. 
```
class RenderState {

  json() {
    return {
      ....,
      'entitlement': this.entitlement.json(),
    }
  }
}
```

Reply:
Well, I was reacting to the fact that we probably can't just pass the plain `Entitlement` object to expression evaluator. It has to be jsonified (i.e. `json()` should be called on it).

Reply:
Done